specs/Checkbox.md

```diff
@@ -316,12 +322,31 @@ Same behavior as above except no preview of toggled state through hover.
 
 ### Screenreader accessibility:
 
+#### `Fluent UI & Fluent UI Northstar comparison`:
+
+From accessibility point of view the main difference is in type of element used for Checkbox itself:
+
+- Fluent UI has `<input>` element with type="checkbox"
+- Fluent UI Northstar has `<div>` element with role="checkbox"
+
+  [Verification with screen reader](https://jurokapsiar.github.io/open-a11y/components/checkbox.research) doens't show any differences between these two approaches, everything was working as expected.
```

Comment:
the difference is that an `input` receives focus and manages keyboard events naturally. If you allow the input to be changed to another type of element (not sure if there is a reason to), you'll need to manage these manually.

I believe `input` is needed for an html form scenario as well. Is there any reason why v0 doesn't use `input`?

Reply:
no specific reason for this case - they both behave good for screen readers and yes, the benfit of using input is that it works out of the box with forms.